lms/djangoapps/discussion_api/views.py

```diff
@@ -84,6 +84,8 @@ class ThreadViewSet(_ViewMixin, DeveloperErrorViewMixin, ViewSet):
 
         * page_size: The number of items per page (default is 10, max is 100)
 
+        * topic_id (optional): The topic, if given, to retrieve the threads for
```

Comment:
I have been specifically calling out required parameters and leaving optional parameters unannotated.